subprojects/docs/src/docs/userguide/buildCache.adoc

```diff
@@ -306,10 +306,31 @@ Gradle pushes build outputs to any build cache that is enabled and has api:org.g
 By default, the local build cache has push enabled, and the remote build cache has push disabled.
 
 The local build cache is pre-configured to be a api:org.gradle.caching.local.DirectoryBuildCache[] and enabled by default.
-The `DirectoryBuildCache` will periodically clean-up the local cache directory to keep it under a configurable target size.
 The remote build cache can be configured by specifying the type of build cache to connect to (api:org.gradle.caching.configuration.BuildCacheConfiguration#remote(java.lang.Class)[]).
 
-Gradle supports connecting to a remote build cache backend via HTTP. This can be configured in `settings.gradle`.
+[[sec:build_cache_configure_local]]
+==== Built-in local build cache
+
+The built-in local build cache, api:org.gradle.caching.local.DirectoryBuildCache[], uses a directory to store build cache artifacts.
+You can configure the directory the local build cache uses to store the build outputs.
+By default, it resides in the Gradle user home directory.
+Gradle will periodically clean-up the local cache directory to keep it under a configurable target size.
+This means that the local build cache directory may temporarily grow over the target size until the next clean-up is scheduled.
+
+For more details on the configuration options refer to the DSL documentation of api:org.gradle.caching.local.DirectoryBuildCache[].
+Here is an example of the configuration.
+
+++++
+<sample id="directoryBuildCacheConfiguration" dir="buildCache/configure-built-in-caches" title="Configure the local cache">
+    <sourcefile file="settings.gradle" snippet="configure-directory-build-cache"/>
+</sample>
+++++
+
+
+[[sec:build_cache_configure_remote]]
+==== Remote HTTP build cache
+
+Gradle supports connecting to a remote build cache backend via HTTP.
```

Comment:
I'd say "Gradle has built-in support for connecting to..."

Reply:
fixed.